### ISSUE TITLE
Behat failing on Travis (again!)

### ISSUE THREAD

**Issue by timhunt** [maths/moodle-qtype_stack#580]
It's currently failing with error:

Selenium server is not running, you need to start it to run tests that involve Javascript. More info in https://docs.moodle.org/dev/Running_acceptance_test

**Comment by timhunt** [comment]
To try to fix this, I copied the configuration from some plugins where it was working. (E.g. https://github.com/timhunt/moodle-mod_attendance/blob/master/.travis.yml) but that did not good.

Then, in displair, I cloned that repository to my github, and ran the tests, and they failed with the same Selenium error that we are seeing here :-(

I guess that Travis are in the middle of deploying some upgrade across there server estate, and at the moment it is affecting some people, but not others. I will leave this for a few days, then come back and try again.

For now, I have made some changes:

* Moodle has just updated Node JS version to 14.2.0, so I made that change.
* I have copied in the PATH/JAVA_HOME magic that is working for other people, in case it helps.
* I have put Behat in the allow_failures section, so we have have a green build even with Behat not working.
